src/generate.js

```diff
@@ -5,7 +5,11 @@ var self = module.exports;
 // Generates a random number
 var randomNumber = function(max) {
 	// gives a number between 0 (inclusive) and max (exclusive)
-	return crypto.randomBytes(1)[0] % max;
+	var rand = crypto.randomBytes(1)[0];
+	while (rand >= max - (256 % max)) {
```

Comment:
```suggestion
	while (rand >= 256 - (256 % max)) {
```

Should it be this instead?

Reply:
Yes, it should be.